app/views/projects/show.html.erb

```diff
@@ -74,11 +74,11 @@
             <strong><%= t("projects.show.project_access_type") %> </strong>
             <%= @project.project_access_type %>
           </p>
-          <% if !@project.description.nil? %>
+          <% if @project.description.present? %>
             <p>
               <strong><%= t("projects.show.project_description") %> </strong>
             </p>
-            <div class="projects-details-description"><%= sanitize @project.description %></div>
+            <div class="projects-details-description"><%= sanitize @project.description.gsub(/\A\s*(<p><br><\/p>)*|(<p><br><\/p>)*\s*\z/, ' ') %></div>
```

Comment:
Here using gsub, we are replacing the interested part with a space. Instead of replacing space, we can replace it with a blank string. That will be better.

Reply:
yeah we can do that too

Reply:
done with this change sir